src/main/java/de/qabel/desktop/util/QuotaUtil.java

```diff
@@ -0,0 +1,21 @@
+package de.qabel.desktop.util;
+
+import de.qabel.core.accounting.QuotaState;
+
+import static humanize.Humanize.binaryPrefix;
+
+public class QuotaUtil {
+
+    public static int ratioByDiff(long usedQuota, long availableQuota) {
+        return (int) (usedQuota / (double) availableQuota * 100);
+    }
+
+    public static String getQuotaDescription(QuotaState q, String freeLabel) {
+        String seperator = " " + freeLabel + " / ";
```

Comment:
This is easier to customize if you use a translation with placeholder. "%s free / %s" or similar. There should already be examples for that.